addon/lib/main.js

```diff
@@ -16,6 +16,7 @@ const helperworker = require("./helperworker");
 const { ToggleButton } = require('sdk/ui/button/toggle');
 const panels = require("sdk/panel");
 const { watchFunction, watchWorker } = require("./errors");
+const user = require("user");
```

Comment:
Maybe the FxA additions belong in a separate `profile` module? Also, since the OAuth code lives elsewhere now, I should revert the changes to this file...